models/comments.model.js

```diff
@@ -14,3 +15,19 @@ exports.selectCommentsByArticleId = (article_id) => {
     return rows;
   });
 };
+
+exports.insertCommentByArticleId = (comment, article_id) => {
+  let querystring = `INSERT INTO comments (body,article_id,author,votes) VALUES ($1,$2,$3,$4) RETURNING *`;
+  const { body, username } = comment;
+  const valuesArray = [body, article_id, username, 0];
+
+  if (typeof body !== "string") {
```

Comment:
If you try and insert an incorrect datatype then  SQL will error for you - you just need to handle this error (and others 👀 ) accordingly